exercises/practice/allergies/.approaches/bitwise-and-case/content.md

```diff
@@ -0,0 +1,78 @@
+# Bitwise and case expression
+
+```elm
+intToAllergen : Int -> Maybe Allergy
+intToAllergen x =
+  case x of
+    1 ->
+        Just Eggs
+    2 ->
+        Just Peanuts
+    4 ->
+        Just Shellfish
+    8 ->
+        Just Strawberries
+    16 ->
+        Just Tomatoes
+    32 ->
+        Just Chocolate
+    64 ->
+        Just Pollen
+    128 ->
+        Just Cats
+    _ ->
+        Nothing
+
+toList : Int -> List Allergy
+toList score =
+  [ 1, 2, 4, 8, 16, 32, 64, 128 ]
+  |> List.filterMap ( \allergenInt -> intToAllergen (Bitwise.and score allergenInt))
+
+isAllergicTo : Allergy -> Int -> Bool
+isAllergicTo allergy score =
+  toList score |> List.member allergy
+```
+
+## In this approach
+
+This approach uses case expressions to represent a bit mask for the allergy.
+
+A hard coded list of all the possible allergy values is created.
+[`List.filterMap`][list-filter-map] is used to filter and map the list.
+It converts the allergy values to their matching `Allergy` or filters them out by returning `Nothing`.
+
+You can see a variation of [a `Bitwise` and `case` expression solution on Exercism][bitwise-and-case-solution].
```

Comment:
That was my suggestion, but I think in the end this represents an approach name, not Elm values
```suggestion
You can see a variation of [a Bitwise and case expression solution on Exercism][bitwise-and-case-solution].
```